src/natcap/invest/internationalization/locales/es/LC_MESSAGES/messages.po

```diff
@@ -1,4010 +1,4903 @@
-msgid "user-defined recharge layer (advanced)"
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:205
-msgid ""
-"Map of local recharge data. Required if User-Defined Local Recharge is "
-"selected."
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:208
-msgid "local recharge"
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:212
-msgid "Use user-defined climate zone data in lieu of a global rain events table."
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:215
-msgid "climate zones (advanced)"
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:222
-msgid ""
-"Climate zone ID numbers, corresponding to the values in the Climate Zones"
-" map."
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:228
-msgid ""
-"The number of rain events that occur in each month in this climate zone. "
-"Replace [MONTH] with the month abbreviations: jan, feb, mar, apr, may, "
-"jun, jul, aug, sep, oct, nov, dec, so that there is a column for each "
-"month."
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:236
-msgid ""
-"Table of monthly precipitation events for each climate zone. Required if "
-"User-Defined Climate Zones is selected."
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:239
-msgid "climate zone table"
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:246
-msgid ""
-"Map of climate zones. All values in this raster must have corresponding "
-"entries in the Climate Zone Table."
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:249
-msgid "climate zone map"
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:253
-msgid "Use montly alpha values instead of a single value for the whole year."
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:256
-msgid "use monthly alpha table (advanced)"
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:264
-msgid "Values are the numbers 1-12 corresponding to each month."
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:271
-msgid "The alpha value for that month."
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:275
-msgid ""
-"Table of alpha values for each month. Required if Use Monthly Alpha Table"
-" is selected."
-msgstr ""
-
-#: src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py:278
-msgid "monthly alpha table"
-msgstr ""
-
+# Spanish translations for InVEST.
+# Copyright (C) 2022 Natural Capital Project
+# This file is distributed under the same license as the InVEST project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: InVEST 3.12\n"
+"Report-Msgid-Bugs-To: jdouglass@stanford.edu\n"

```

Comment:
I don't recall why my email is here ... @emlys would you like this to be your email?  Or maybe we should have it email the software team email list, or else an alias at `naturalcapitalproject.org`?